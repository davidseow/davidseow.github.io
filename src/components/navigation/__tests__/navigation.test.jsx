import { render } from "@testing-library/react";
import Navigation from "../navigation";

describe("Navigation", () => {
  it("should render", () => {
    const { asFragment } = render(<Navigation />);
    expect(asFragment()).toMatchSnapshot();
  });
});
