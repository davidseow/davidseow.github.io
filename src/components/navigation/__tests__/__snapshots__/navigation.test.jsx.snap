// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render 1`] = `
<DocumentFragment>
  <navigation>
    <ul>
      <li>
        <a
          href="https://www.instagram.com/davidseow/"
        >
          Instagram
        </a>
      </li>
      <li>
        <a
          href="https://github.com/davidseow"
        >
          Github
        </a>
      </li>
      <li>
        <a
          href="https://www.linkedin.com/in/seowdavid/"
        >
          LinkedIn
        </a>
      </li>
    </ul>
  </navigation>
</DocumentFragment>
`;
